mods/ra/maps/siberian-conflict-1-fresh-tracks/rules.yaml

```diff
@@ -0,0 +1,71 @@
+World:
+	LuaScript:
+		Scripts: fresh-tracks.lua
+	MissionData:
+		WinVideo: sovbatl.vqa
+		LossVideo: overrun.vqa
+		Briefing: Soviet activity in Siberia is increasing. A large shipment of atomic material is on its way through this area, and we need it stopped. Destroy all of the Soviet convoy trucks and prevent the Soviets from furthering their atomic testing.\n\nBe careful of the convoy truck cargo -- atomic material is highly unstable.\n\nThe mission is complete when all the convoy trucks from each of the seven convoys have been destroyed.
```

Comment:
This will be a problem: The briefing explicitly says "seven convoys". But it should be easy to resolve by just removing "seven" here (the sentence is still ok without it imo).